docs/rfcs/README.md

```diff
@@ -0,0 +1,47 @@
+# The RFC Process
+
+An RFC (**R**equest **F**or **C**omments) is a process through which contributors can solicit buy-in
+for proposed changes to the codebase and repository at-large. It was introduced in the first RFC,
+[0001-rfc-process](./0001-rfc-process.md), which includes additional details about the reasoning
+for including the process. 
+
+This is an overview of what kind of changes benefit from or require the consensus-building that the 
+RFC process provides, as well as a brief guide on how to draft them.
+
+## What kinds of changes require an RFC?
+
+1. Substantial changes to public APIs in Android Components, like the changes found in [0003 Concept Base Component](./0003-concept-base-component.md) and [0008 Tab Groups](docs/rfcs/0008-tab-groups.md). 
+2. Changes to process that affect other teams, like the changes found in [0001 RFC Process](./0001-rfc-process.md), [0013 Add stakeholders to RFCs](./0013-rfc-process-updates.md), and [0007 Synchronized Releases](./0007-synchronized-releases.md).
+3. Proposals for changes to areas of the codebase that are owned by CODEOWNERS outside the author's team.
+
+## What kind of other changes can an RFC be useful for?
+
+1. Announcing a rough plan for changes to a public API you own in order to solicit feedback.
+2. Soliciting feedback for architectural changes that affect the entire codebase, like [0009 Remove Interactors and Controllers](./0009-remove-interactors-and-controllers.md).
+
+## How to contribute an RFC
+
+There is a [template](./0000-template.md) that can be useful for structure. While drafting
```

Comment:
> While drafting a proposal, consider the scope of your changes. Generally, the higher the impact the changes will have would be things like example use cases, prototypes, and considered alternatives.

on downstream consumers of APIs, other teams, or users the more detail the RFC should include. Detail
In an effort to keep this README concise, I wonder if this information is adding anything the new template isn't? Or perhaps this should move to the template?

Reply:
I think this is useful general advice, and that it would probably need to be repeated in a few sections in the template so I've kept it here for now but tried to streamline it a bit. LMK if you feel strongly it should change